app/assets/javascripts/components/course.cjsx

```diff
@@ -36,7 +36,7 @@ Course = React.createClass(
     @state.current_user
   submit: (e) ->
     e.preventDefault()
-    return unless confirm "Upon submission, this course will be automatically posted to Wikipedia with your user account. After that, new edits to the timeline will be mirrored to Wikipedia.\n\nAre you sure you want to do this?"
+    return unless confirm #{I18n.t("courses.warn_mirrored")}
```

Comment:
This is keeping the specs from passing, because it doesn't generate the confirmation dialog that is part of the course creation flow. Changing that line to `return unless confirm I18n.t("courses.warn_mirrored")` should fix it.